app/2.0/start/toolbox/deploy.md

```diff
@@ -58,36 +80,38 @@ and create a new project
 
 1.  `cd` into the main folder for your app (e.g. `my-app/`).
 
-1. Create an `app.yaml` file with the following contents:
+1. Create an `app.yaml` file with the following contents (replace <code><var>build-folder-name</var></code> with the name of your build folder):
```

Comment:
I think both of this example and the firebase one should use `es5-bundled` throughout. Neither of the other options will run on all browsers. 

You really want a custom, dynamic serving solution (like `prpl-server-node`) if you're going to serve either of the other builds.